src/cli.ts

```diff
@@ -9,6 +9,8 @@ async function run(argv) {
     .brand('airfoil')
     .src(__dirname)
     .plugins('./node_modules', { matching: 'airfoil-*', hidden: true })
+    .version()
```

Comment:
This overrides the `version` command. Could we remove this?

Test it out for yourself:

```
airfoil version patch
```

The above will now just print out the airfoil version.

`airfoil --help` includes the current version

Reply:
removed ✅